cloudify-manager-worker/README.md

```diff
@@ -203,6 +203,24 @@ Apply created config map:
 $ kubectl apply -f license.yaml
 ```
 
+### Create secret for okta-license - required if using Okta/SSO
```

Comment:
@ahmadiesa-abu Thanks for the PR, it looks good.
Let's just place it in the right location - that'll be towards the bottom underneath 'ingress' / above 'troubleshooting'
Also, please do the same in the file readme.md.gotmpl

Reply:
@yuvlar addressed, thanks for the input